.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,3 +30,5 @@ What types of changes does your code introduce to Graphback?
 ## Further comments
 
 <!--If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...-->
+
+<!--Note: The template gives you full control over the content of your pull request. Feel free to remove the parts you feel are irrelevant to your particular pull request.-->
```

Comment:
We still want people to use this template, so the following suggests keeping it as close as possible to original:

```suggestion
<!--The content below is a suggested layout for writing your pull request description. You may extend or remove parts you deem relevant or irrelevant to your changes. -->
```